test/api/repositories/template-inputs/001-inputs/expected-response.json

```diff
@@ -26,5 +26,27 @@
       ]
     }
   ],
-  "preconfiguredSteps": []
+  "initialState": {
+    "valid": false,
+    "stepGroups": [
+      {
+        "id": "g01",
+        "valid": false,
+        "error": "All steps (1) must be selected.",
+        "steps": [
+          {
+            "id": "g01-s01",
+            "configured": false,
+            "valid": true,
+            "inputs": [
+              {
+                "id": "shopify-token",
+                "visible": true
+              }
+            ]
+          }
+        ]
+      }
+    ]
+  }
```

Comment:
AdHoc/ASAP task od UI.

Zistilo sa, že ked UI dostane index `inputs`, tak im nestačia `preconfiguredSteps` ale potrebuju `initialState`, ktory je totožný s tým, ak by zavolali `validate inputs` s default hodnotami (<<< da sa to pochopiť? :thinking: ).

Tj. teraz tam budú mať informáciu na viac, či je daný `step`/`stepGroup` `valid`, keďže šablóna sa mohla upraviť, mohol pribudnúť nový input, a step môže byť `configured=true, valid=false`.

Zmena sa tyka `upgrade/inputs` aj `use/inputs`, keďže majú rovnaký result a aj v UI kóde im to potom stačí mať iba raz (aj ked pri `use/inputs` by sme to nemuseli vratit, lebo tam je všetko `configured=false`).

Reply:
jj. rozumím